### DISCUSSION TITLE
How to use `keepMounted` with WindowVirtualizer?

### DISCUSSION THREAD

**Discussion by hungnb31** [inokawa/virtua#577] [Q&A] [answered]
I can see `keepMounted` only available with `Virtualizer` and `VList`, but not `WindowVirtualizer`. Any idea to add it to `WindowVirtualizer` or do we have the other way in `WindowVirtualizer` to achieve the same thing?

**Comment by inokawa** [comment, marked as answer]
WindowVirtualizer doesn't have `keepMounted` prop just because it wasn't requested until now. I think it's ok to implement it to WindowVirtualizer (PR welcome).
